app/assets/javascripts/models/beta/story.js

```diff
@@ -22,8 +22,8 @@ export const needConfirmation = story =>
   story.state === status.RELEASE
 
 export const comparePosition = (a, b) => {
-  const positionA = parseFloat(a.position);
-  const positionB = parseFloat(b.position);
+  const positionA = parseInt(a.newPosition);
```

Comment:
Why do we need to receive these as strings? :thinking:

Reply:
As I said earlier, for some reason the `position` property is received in a string format, but `newPosition` is not, it has already been fixed.